src/Component/Menu/Navigation/Navigation.tsx

```diff
@@ -55,15 +58,61 @@ export const Navigation: React.FC<NavigationProps> = ({
 
   const navigationConf = config.navigation;
 
-  const navigationContentChildren: ItemType[] = entityConfigs.map(conf => ({
-    key: conf.entityType,
-    label: (
-      <>
-        <BankOutlined />
-        <span>{TranslationUtil.getTranslationFromConfig(conf.navigationTitle, conf.i18n)}</span>
-      </>
-    )
-  }));
+  const navigationContentChildren: ItemType[] = [];
+
+  if (entityConfigs && entityConfigs.length > 0) {
+
+    if (navigationConf?.general?.imagefiles?.visible) {
+      const applicationConfig = entityConfigs.find(e => e.entityType === 'application');
+      const layersConfig = entityConfigs.find(e => e.entityType === 'layer');
+      const userConfig = entityConfigs.find(e => e.entityType === 'user');
+      const groupsConfig = entityConfigs.find(e => e.entityType === 'group');
```

Comment:
does it make sense that applications, layers etc. are only visible if imagefiles are visible?

Reply:
I think you might need additional checks for these, or apply the if-condition to imagefiles only

Reply:
I think yes since if the menu is collapsed there is no need to show the configs. Nonetheless I am open for suggestions!

Reply:
After testing I have come to the conclusion, that if `navigationConf.general.imagefiles.visible` is somehow false, then the application tab is invisible and by that its sub-points are also invisible. If a whole tab is gone then something went completely wrong elsewhere since the application tab must be visible. Therefore I'd say there is no reason for further checks (of course as far as I can see through my testing).

Reply:
that doesn't seem very logical to me

currently, this will do the following: if imagefiles are not visible, the menu items for applications, layers etc. are not added to the menu

what i would expect: if imagefiles are not visible, the item should be visible regardless. I think you just need to remove the if condition completely here.

Reply:
Done!